.github/workflows/ci.yml

```diff
@@ -8,10 +8,15 @@ on:
 jobs:
   test:
     runs-on: ${{ contains( github.event.pull_request.labels.*.name, 'use_fast_ci') && 'ubuntu-latest-8core' || 'ubuntu-latest' }}
```

Comment:
We never want to run lint on ubuntu-latest-8core. We've not used this in a while, and I almost took in out in my PR, but I wasn't touching this file. We should either remove this now, or modify it so it only applies to "specs" and not "lint".

Reply:
```suggestion
    runs-on: ubuntu-latest
```